gitops/snapshot.go

```diff
@@ -76,6 +76,15 @@ const (
 	// SnapshotStatusReportAnnotation contains metadata of tests related to status reporting to git provider
 	SnapshotStatusReportAnnotation = "test.appstudio.openshift.io/git-reporter-status"
 
+	// SnapshotPRGroupAnnotation contains the pr group name
+	SnapshotPRGroupAnnotation = "test.appstudio.openshift.io/pr-group"
+
+	// SnapshotPRGroupHashLabel contains the pr group name in sha format
+	SnapshotPRGroupHashLabel = "test.appstudio.openshift.io/pr-group-sha"
+
+	// BuildPipelineRunStartTime contains the start time of build pipelineRun
+	BuildPipelineRunStartTime = "test.appstudio.openshift.io/pipelinerunstarttime"
```

Comment:
I notice we have had label [test.appstudio.openshift.io/pipelinerunfinishtime](https://github.com/konflux-ci/integration-service/blob/main/gitops/snapshot.go#L83) for build PLR completionTime, then to be consistent, let's use"test.appstudio.openshift.io/pipelinerunstarttime" as the start time annotation  of build PLR. HDYT?

Reply:
Yes, that makes sense to me.

Reply:
Thanks!